### DISCUSSION TITLE
Why is there a HexSingleGrid(HexGrid) and a HexGrid?

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#1827] [Q&A] [answered]
Why is the HexSingleGrid subclassed from HexGrid and HexGrid has nothing in it? 
I was reading through the code and confused by this -- because it is also visible in the docs. Wondering if there was logical reason or if someone remembered why? I was thinking we should get rid of HexGrid or make it private. 

See the following two lines -https://github.com/projectmesa/mesa/blob/b2f642fd1a986f3114ba6f6f6e9bc119c037062f/mesa/space.py#L742
https://github.com/projectmesa/mesa/blob/b2f642fd1a986f3114ba6f6f6e9bc119c037062f/mesa/space.py#L768

**Comment by rht** [comment, marked as answer]
It wasn't made private in order to maintain backward compatibility, but should have been made private during the 2.0 release.

**Comment by jackiekazil** [comment]
Thank you. Where do you think is the best place for marking that as something for 3.0? 
Who knows when 3.0 happens... but I want to make sure that doesn't get forgotten about.

**Reply by rht** [reply]
We could open an issue that is set as a blocker for 3.0 milestone.

**Comment by jackiekazil** [comment]
https://github.com/projectmesa/mesa/issues/1847
